src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/StandardCommands.cs

```diff
@@ -4,409 +4,329 @@
+        /// Gets the GUID/integer value pair for the SizeToControlHeight command. Read only.
         /// </summary>
         public static readonly CommandID SizeToControlHeight = new CommandID(s_standardCommandSet, VSStandardCommands.cmdidSizeToControlHeight);
+        
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the SizeToControlWidth command. Read
-        ///       only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the SizeToControlWidth command. Read only.
         /// </summary>
         public static readonly CommandID SizeToControlWidth = new CommandID(s_standardCommandSet, VSStandardCommands.cmdidSizeToControlWidth);
+        
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the SizeToFit command. Read only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the SizeToFit command. Read only.
         /// </summary>
         public static readonly CommandID SizeToFit = new CommandID(s_standardCommandSet, VSStandardCommands.cmdidSizeToFit);
+
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the SizeToGrid command. Read only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the SizeToGrid command. Read only.
         /// </summary>
         public static readonly CommandID SizeToGrid = new CommandID(s_standardCommandSet, VSStandardCommands.cmdidSizeToGrid);
+
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the SnapToGrid command. Read only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the SnapToGrid command. Read only.
         /// </summary>
         public static readonly CommandID SnapToGrid = new CommandID(s_standardCommandSet, VSStandardCommands.cmdidSnapToGrid);
+
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the TabOrder command. Read only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the TabOrder command. Read only.
         /// </summary>
         public static readonly CommandID TabOrder = new CommandID(s_standardCommandSet, VSStandardCommands.cmdidTabOrder);
+
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the Undo command. Read only.
-        ///    </para>
+        /// Gets the GUID/integer value pair for the Undo command. Read only.
         /// </summary>
         public static readonly CommandID Undo = new CommandID(s_standardCommandSet, VSStandardCommands.cmdidUndo);
+
         /// <summary>
-        ///    <para>
-        ///       Gets the GUID/integer value pair for the MultiLevelUndo command. Read
-        ///       only.
-        ///    </para>
-        /// </summary>
+        /// Gets the GUID/integer value pair for the MultiLevelUndo command. Read only.
```

Comment:
You deleted the closing `</summary>`.  Since you're changing the XML comments, can you validate that everything compiles successfully when you turn on the validation in the compiler?

Reply:
Fixed. I'm not sure how to turn on the validation?

Reply:
By turning on XML doc file generation, e.g.
https://github.com/dotnet/corefx/blob/ba496ae6a9ae055708b630b3f73395a76c2ce6ce/src/System.Threading.Tasks.Extensions/src/System.Threading.Tasks.Extensions.csproj#L6

Reply:
I hadn't realized that we don't have this enabled for all our assemblies. We probably should enable it globally - grandfathering in any that have invalid XML currently.

Reply:
ie stick it in the root dir.props

Reply:
We would need to grandfather-in pretty much every assembly, as it'll warn-as-error for public types/members missing XML comments.

Reply:
I turned this on for this library in this PR (I had to silence warnings for no documentation for public types etc.)

This PR is already pretty massive so I'll do that in another PR